### PR TITLE
aii-opennebula: include support to set different RPC cloud domains

### LINKED ISSUE #142: aii-opennebula: include support to set different cloud domains into /etc/aii/opennebula.conf
At the moment ONE AII uses `/etc/aii/opennebula.conf` to access to a single RPC endpoint:
```cfg
[rpc]
password=
user=
```
Include support to be able to use different cloud domains if it is required, as example:
```cfg
[rpc]
...
[DOMAIN]
...
```
If `[DOMAIN]` is not set AII should still use the `[rpc]` section to keep backwards compatibility.

### PR DESCRIPTION
**Pull request by alvarosimon**
* Fix #142 : include support to set different cloud domains into /etc/aii/opennebula.conf

### CODE REVIEW

File: aii-opennebula/src/main/perl/opennebula.pm [hunk 1]
```diff
@@ -38,12 +39,16 @@ sub make_one
...
+    my $domainname = $data->getElement (DOMAINNAME)->getValue;
 
     $config = Config::Tiny->read($filename);
-    my $port = $config->{rpc}->{port} || 2633;
-    my $host = $config->{rpc}->{host} || "localhost";
-    my $user = $config->{rpc}->{user} || "oneadmin";
-    my $password = $config->{rpc}->{password};
+    if (exists($config->{$domainname})) {
+        $rpc = $domainname;
+    };
```
Comment (stdweird) [inline aii-opennebula/src/main/perl/opennebula.pm]: add a log message indictae which section was used

File: aii-opennebula/src/main/perl/opennebula.pod [hunk 2]
```diff
@@ -116,6 +118,40 @@ Following package dependencies should be installed to run the AII:
...
 
+Set OpenNebula endpoints RPC connector /etc/aii/opennebula.conf
+
+It must include at least one RPC endpoint and password.
+
+By default ONE AII uses oneadmin user.
+
+It is also possible to set a different endpoint for each VM domain, as example:
+
+=over
```
Comment (stdweird) [inline aii-opennebula/src/main/perl/opennebula.pod]: don't use a pod list here, just a raw text field. indent eacch line with 4 spaces and this should work

**Comment by alvarosimon** [comment]
@stdweird fixed remarks. I have tested this in gastly using shuppet VMs.

**Comment by jrha** [comment]
LGTM @apdibbo?

**Comment by apdibbo** [comment]
LGTM

**Pull request merged**